includes/utilities.inc

```diff
@@ -364,7 +364,7 @@ function islandora_solr_prepare_solr_results($solr_results) {
       }
 
       // Implode.
-      $value = is_array($value) ? implode(", ", $value) : $value;
+      $value = is_array($value) ? implode(variable_get('islandora_solr_search_field_value_separator', ', '), $value) : $value;
```

Comment:
Actually as this is altering the returned values from Solr perhaps this shouldn't be altered at all and only the results output to the screen should be affected?